docs/architecture.md

```diff
@@ -21,52 +21,52 @@ This leads to constant development and DevOps/MLOps work.
-Imagine a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
+Imagine having a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
 A world of easily developed, published, or consumed data or ML "functions" that can be used to form complex and large-scale offline or real-time ML pipelines.
 
-In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code reuse across projects and companies and thus further accelerate your work.
+In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code for reuse across projects and companies and thus further accelerate your work.
 
-<b>This is the goal of MLRun, Simplify & Accelerate Time To Production.</b>
+<b>This is the goal of MLRun - Simplify & Accelerate Time To Production.</b>
 
 ## Architecture 
 
 <img src="_static/images/mlrun-architecture.png" alt="mlrun-architecture" width="800"/>
 
-MLRun comprises of the following layers:
+MLRun comprises the following layers:
 * **Feature & Artifact Store** - Handle the ingestion, processing, metadata and storage of data and features across multiple repositories and technologies
 * **Elastic Serverless Runtimes** - Convert simple code to scalable and managed micro-services with 
-  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  .
+  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  
 * **ML Pipeline Automation** - Automated data preparation, model training & testing, 
-  deployment of production (real-time) pipelines, and end to end monitoring.
+  deployment of production (real-time) pipelines, and end-to-end monitoring
 * **Central Management** - Unified portal, UI, CLI, and SDK to manage the entire MLOps workflow which is 
-  accessible from everywhere. 
+  accessible everywhere
 
 <a id="basic-components"></a>
 ## Basic Components
 
 MLRun has the following main components that are used throughout the system:
 
-- <a id="def-project"></a>**Project** &mdash; a container for organizing all of your work on a particular activity.
-    Projects consist of metadata, source code, workflows, data and artifacts, models, triggers, and member management for user collaboration.
+- <a id="def-project"></a>**Project** &mdash; a container for organizing all of your work on a particular activity;
+    Projects consist of metadata, source code, workflows, data and artifacts, models, triggers, and member management for user collaboration
```

Comment:
REJECT: Please revert this change. We intentionally used multiple sentences for some list items, to avoid very long sentences (semicolons don't make it multiple sentences). For consistency, we used periods at the end of the other list items as well. It's OK to use periods at the end of list items (especially in technical documentation) even if the items aren't exactly phrased as full sentences.
Also note that had we kept the semicolon, the subsequent text should have been in lowercase and not capitalized ("Projects" > "projects").